packages/clay-pagination/src/__tests__/index.tsx

```diff
@@ -118,7 +118,10 @@ describe('ClayPagination', () => {
 
 		fireEvent.click(getByText(container, '...') as HTMLElement, {});
 
-		fireEvent.click(getByText(document.body, '4') as HTMLElement, {});
+		fireEvent.click(
```

Comment:
Out of curiosity, why did you change that?

Reply:
Source formatting

Reply:
Source formatting